draft-ietf-rats-architecture.md

```diff
@@ -316,6 +316,26 @@ Relying Party:
 : A remote server that will securely grant the Attester
 permission to continue operating (i.e., not reboot) for a period of time
 
+## FIDO Biometric Authentication
+
+In the FIDO protocol, it is the device in the user’s hand that authenticates the human user by biometrics like fingerprints or PIN and password.
```

Comment:
(simplify grammar)
```suggestion
In the Fast IDentity Online (FIDO) protocol, the device in the user’s hand authenticates the human user by biometrics such as fingerprints or PIN and password.
```

Also can we add a citation for the protocol?

Reply:
Fine with the change. I think that this is *the* most important value add of FIDO: the Authenticator component that can require a human to be in the loop.